web/styles/buttons.css

```diff
@@ -0,0 +1,186 @@
+.action-button {
+    /* Reset button styles — borrowed from TailwindCSS Preflight */
+    font-family: inherit;
+    font-feature-settings: inherit;
+    font-variation-settings: inherit;
+    font-size: 100%;
+    margin: 0;
+    text-transform: none;
+    appearance: button;
+    background-image: none;
+    cursor: pointer;
+
+    display: flex;
+    gap: 0.5ch;
+    justify-content: center;
+    align-items: center;
+    line-height: 1.3333;
+    font-weight: 550;
+    letter-spacing: 0.02ch;
+    padding: 0.2667em 0.6667em;
+    color: var(--color-btn-text, --color-neutral-secondary-btn-text);
+    background-color: var(--color-btn-bg, --color-neutral-secondary-btn-bg);
+    border-radius: 4px;
+    white-space: nowrap;
+    user-select: none;
+    border: none;
+}
+
+.action-button__label {
```

Comment:
Why not call this `action-button-label`? We're not really using underscores elsewhere in these style names.

Reply:
Yep, I missed this one, will fix it in the next push.